packages/backend-common/config.d.ts

```diff
@@ -20,6 +20,16 @@ export interface Config {
   };
 
   backend: {
+    /** Backend configuration for when request authentication is enabled */
+    authorization?: {
+      /**
+       * Secret shared by all backends for generating tokens
+       * Format is base64 24-bit key
+       * @visibility secret
+       */
+      secret?: string;
+    };
```

Comment:
Could make `secret` required tbh, as in if the `authorization` block is provided then you need to specify a secret

Reply:
:+1: done in db1befcf39